compose/docker-compose.yml

```diff
@@ -70,20 +70,54 @@ services:
     networks:
       - kong-net
     ports:
-      - "8000:8000/tcp"
+      # FIXME: the following two environment variables default to an insecure value (0.0.0.0)
+      # according to the CIS Security test.
+      - "${KONG_INBOUND_PROXY_LISTEN:-0.0.0.0}:8000:8000/tcp"
+      - "${KONG_INBOUND_SSL_PROXY_LISTEN:-0.0.0.0}:8443:8443/tcp"
+      # Consider either making them mandatory but undefined, like so (which would be backwards-breaking):
+      # - "${KONG_INBOUND_PROXY_LISTEN?Missing inbound proxy host}:8000:8000/tcp"
+      # - "${KONG_INBOUND_SSL_PROXY_LISTEN?Missing inbound proxy ssl host}:8443:8443/tcp"
+      # Alternative is deactivating check 5.13 in the security bench, if we consider Kong's own config to be enough security here
+
       - "127.0.0.1:8001:8001/tcp"
-      - "8443:8443/tcp"
       - "127.0.0.1:8444:8444/tcp"
     healthcheck:
       test: ["CMD", "kong", "health"]
       interval: 10s
       timeout: 10s
       retries: 10
-    restart: on-failure
+    restart: on-failure:5
+
+
+    # Hardcoded because Docker-compose cant't handle variable interpolations for integer values
+    # see https://github.com/docker/compose/issues/8297
+    cpu_shares: 1024
```

Comment:
```suggestion
```

This is applicable to docker-compose version 2 only and is the current build failure

```
Server: openresty

Compose file contains unsupported options:

cpu_shares: Set resource limits using deploy.resources
```